src/likecounter.test.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import likeCounter from './likecounter.js';
+
+describe('likeCounter', () => {
```

Comment:
- [ Required ] Good job adding tests for the `likescounter` module, however, I think it would be better if you make sure that you have tests for all edge cases e.g think at least about the scenario in which there are 0 likes on an item (counter function should return 0)

## Example code
```JavaScript
/**
 * @jest-environment jsdom
 */

import likeCounter from './likecounter.js';

describe('likeCounter', () => {
  beforeEach(() => {
    document.body.innerHTML = `
      <button id="testButton">Likes: <span id="likeCount">0</span></button>
    `;
  });

  afterEach(() => {
    document.body.innerHTML = '';
  });

  it('likes counter should return zero likes', () => {
    const buttonId = 'testButton';
    const likesData = {
      testButton: 0,
    };

    const button = document.getElementById(buttonId);

    const updatedLikeCount = likeCounter(buttonId, likesData);

    expect(updatedLikeCount).toBe(0);
    expect(button.innerHTML).toBe('0');
  });
});
```